.travis.yml

```diff
@@ -8,20 +8,14 @@ language: python
 # NOTE: Explicit Python versions make Travis job description more informative
 matrix:
   include:
-    - env: TOX_ENV=py27
-      python: "2.7"
-    - env: TOX_ENV=py34
-      python: "3.4"
-    - env: TOX_ENV=py35
```

Comment:
This commit also removes support for Python 3.5, not only Python 2

Reply:
I have changed to commit message